src/query/plan/operator.cpp

```diff
@@ -4657,23 +4655,12 @@ void CallCustomProcedure(const std::string_view fully_qualified_procedure_name,
   if (memory_limit) {
     SPDLOG_INFO("Running '{}' with memory limit of {}", fully_qualified_procedure_name,
                 utils::GetReadableSize(*memory_limit));
-    utils::LimitedMemoryResource limited_mem(memory, *memory_limit);
-    mgp_memory proc_memory{&limited_mem};
```

Comment:
Since we are tracking memory via extent hooks, I removed that part on the limited memory resource